sdk/storage/azure-storage-blob/tests/test_get_blob_async.py

```diff
@@ -161,11 +163,36 @@ async def test_get_blob_to_bytes_async(self, resource_group, location, storage_a
         blob = self.bsc.get_blob_client(self.container_name, self.byte_blob)
 
         # Act
-        content = await (await blob.download_blob(max_concurrency=2)).readall()
+        content = await (await blob.download_blob()).readall()
 
         # Assert
         self.assertEqual(self.byte_data, content)
 
+    @GlobalStorageAccountPreparer()
+    @AsyncStorageTestCase.await_prepared_test
+    async def test_get_blob_in_raw_stream_async(self, resource_group, location, storage_account, storage_account_key):
+        # parallel tests introduce random order of requests, can only run live
+        await self._setup(storage_account, storage_account_key)
+        blob_name = self._get_blob_reference()
+        blob = self.bsc.get_blob_client(self.container_name, blob_name)
+        file_path = "C:/Users/xiafu/Downloads/testgzip.txt.gz"
+        data = self.get_random_bytes(1024*1024*1024)
+
+        with gzip.open(file_path, 'wb') as stream2:
+            stream2.write(data)
+
+        with open("C:/Users/xiafu/Downloads/testthrottle.txt.gz", 'rb') as stream:
+            data = stream.read()
+            await blob.upload_blob(data, content_settings=ContentSettings(content_encoding='gzip'), overwrite=True)
+
+        # Act
+        # with self.assertRaises()
+
+        content = await (await blob.download_blob(offset=1024, length=512, decompress=False)).readall()
```

Comment:
this is a more specific case, we can focus on sync test_get_blob_in_raw_stream in test_get_blob.py first